charts/tidb-drainer/values.yaml

```diff
@@ -28,7 +28,7 @@ storage: 10Gi
 # disbale detect causality
 disableDetect: false
 # if drainer donesn't have checkpoint, use initial commitTS to initial checkpoint
-initialCommitTs: 0
+initialCommitTs: "0"
```

Comment:
From the comment of drainer dev, operator should not set the default value or set it to "-1", which is the default value in drainer, so we can set it to `# initialCommitTs: "-1"`?
BTW, the `_start_drainer.sh.tpl` also needs to update?

Reply:
_start_drainer.sh.tpl don't need to update.

Reply:
We'd better set the default value in `_start_drainer.sh.tpl` to `-1`.